vendor/priors/gwt.js

```diff
@@ -75,7 +66,11 @@ function postProm(url, data) {
 		resolve(new Int8Array(req.response));
             }
             else {
-		reject(req.getResponseHeader("Trailer"));
+		try {
+		    reject(req.getResponseHeader("Trailer"));
+		} catch (e) {
+		    reject("Error");
```

Comment:
how about adding the actual exception to the error message?

Reply:
done